client/js/app.modules/main/main-page-info.factory.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Name as service not factory, cause factory is more like creation approach but not the return value.